Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift

```diff
@@ -0,0 +1,139 @@
+                
+                HStack {
+                    Spacer()
+                    HStack {
+                        Text("Distance:")
+                        Text(cameraDistance, format: .measurement(
+                            width: .narrow,
+                            usage: .asProvided,
+                            numberFormatStyle: .number.precision(.fractionLength(0))
+                        ))
+                    }
+                    .padding()
+                    .background(.ultraThinMaterial)
+                    .cornerRadius(10)
+                    .shadow(radius: 3)
+                }
+                .padding(.trailing, 8)
+            }
+        }
+        .task {
+            for await newDistance in model.cameraController.$cameraDistance {
+                cameraDistance.value = newDistance
+            }
+        }
+    }
+}
+
+private extension StylePointWithDistanceCompositeSceneSymbolView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A scene with an imagery basemap and world elevation surface.
+        let scene: ArcGIS.Scene = {
+            let scene = Scene(basemapStyle: .arcGISImagery)
+            let elevationSource = ArcGISTiledElevationSource(url: .worldElevationService)
+            scene.baseSurface.addElevationSource(elevationSource)
+            return scene
+        }()
+        
+        /// The camera controller focused on the plane graphic.
+        private(set) var cameraController: OrbitGeoElementCameraController!
+        
+        /// The graphics overlay for the plane graphic.
+        let graphicsOverlay: GraphicsOverlay = {
+            let graphicsOverlay = GraphicsOverlay()
+            graphicsOverlay.sceneProperties.surfacePlacement = .relative
+            return graphicsOverlay
+        }()
+        
+        /// The plane graphic created from a distance composite symbol.
+        private let planeGraphic: Graphic = {
+            // Create the different symbols.
+            let planeSymbol = ModelSceneSymbol(url: .bristol, scale: 100)
+            let coneSymbol = SimpleMarkerSceneSymbol.cone(color: .red, diameter: 200, height: 600)
+            coneSymbol.pitch = -90.0
+            let circleSymbol = SimpleMarkerSymbol(style: .circle, color: .red, size: 10)
+            
+            // Create a distance composite symbol using the symbols.
+            let distanceCompositeSymbol = DistanceCompositeSceneSymbol()
+            distanceCompositeSymbol.addRange(
+                DistanceSymbolRange(symbol: planeSymbol, minDistance: 0, maxDistance: 10000)
+            )
+            distanceCompositeSymbol.addRange(
+                DistanceSymbolRange(symbol: coneSymbol, minDistance: 10001, maxDistance: 30000)
+            )
+            distanceCompositeSymbol.addRange(
+                DistanceSymbolRange(symbol: circleSymbol, minDistance: 30001, maxDistance: 0)
+            )
```

Comment:
Just a suggestion here, take or leave as desired

```suggestion
            distanceCompositeSymbol.addRange(
                DistanceSymbolRange(symbol: planeSymbol, maxDistance: 10000)
            )
            distanceCompositeSymbol.addRange(
                DistanceSymbolRange(symbol: coneSymbol, minDistance: 10001, maxDistance: 30000)
            )
            distanceCompositeSymbol.addRange(
                DistanceSymbolRange(symbol: circleSymbol, minDistance: 30001)
            )
```